.github/ISSUE_TEMPLATE/request_help.yml

```diff
@@ -1,35 +1,36 @@
-name: "Request help"
-description: Please use this template for requesting help.
-title: "request help: "
+name: "Request Help"
+description: Stuck? Ask for help!
+title: "help request: "
 body:
   - type: markdown
     attributes:
       value: |
-        The more information you fill in, the better we can help you.
+        _The more information you share, the faster we can help you._
 
-        - Please use English on public channel.
-        - Encourage using words rather than pictures to describe errors.
-        - Encourage pre-searching with keywords in the issue.
+        Prior to opening the issue, please make sure that you:
+
+        - Use English to communicate.
+        - Search the [open issues](https://github.com/apache/apisix/issues) and [discussion forum](https://github.com/apache/apisix/discussions) to avoid duplicating the issue.
 
   - type: textarea
     id: description
     attributes:
-      label: Issue description
-      description: Provide a clear and concise description of the help you need.
+      label: Description
+      description: Describe the issue you are facing and what you need help with.
     validations:
       required: true
   - type: textarea
     id: environment
     attributes:
       label: Environment
-      description: Request help without environment information will be ignored or closed.
+      description: Share your environment details. Reports without proper environment details will likely be closed.
       value: |
-        - apisix version (cmd: `apisix version`):
-        - OS (cmd: `uname -a`):
-        - OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
-        - etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
```

Comment:
Why remove the `get the info from server-info API`?

Reply:
`etcd version` implies that you have to get it. And the API is self documented in my opinion and it is obvious what it does. So, an additional sentence might be redundant? What do you think? Should I add back that sentence?

Reply:
I see.